package.json

```diff
@@ -199,6 +200,7 @@
     "test": "NODE_ENV=test LOG_LEVEL=silent mocha './app/**/*.test.js' './common/**/*.test.js' './config/**/*.test.js'",
     "test-e2e:chrome": "testcafe chrome test/e2e/*.test.js",
     "test-e2e:ci": "npm run test-e2e:chrome -- --color --reporter spec,xunit:reports/testcafe/results-chrome.xml",
+    "test-e2e:ie": "testcafe 'browserstack:Chrome@53.0:Windows 10' test/e2e/*.test.js",
```

Comment:
Could also use `test-e2e:browserstack`? Then it allows us to extend the test suite once we have it as part of the CI pipeline.

Reply:
Good point 👍